kapitan/inputs/helm/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+cd $(dirname "$0")
+go build -buildmode=c-shared -o libtemplate.so template.go
```

Comment:
It's good that you have this buiild script. 
I'd also suggest you add a step in the `Makefile` in the root directory to call this build script as well. 
Something like `make build_helm_bindings` maybe ? 
That way we can add it to .travis.yml and also test continuously whether the .so lib builds fine or not (?).

Reply:
Yes this is exactly what got me thinking about when we should run this script to build from source. For testing as well, do we require the kapitan developers to have go available so that when they run `make test` the build is triggered such that we can test the helm bindings?

Reply:
Don't add this under `make test`. We can easily have another step, called `build_binding` and run that only on Travis (by adding it to .travis.yml explicitly). That way most developers who won't touch the bindings don't need to install unnecessary dependencies. But at the same time we also have continuous testing to make sure we catch regressions in that build. :)